.github/workflows/CI.yml

```diff
@@ -30,9 +30,15 @@ jobs:
       
       - name: Update mirflowz env with root packages
         run: mamba env update -n mirflowz -f environment.root.yml
+      
+      - name: Update mirflowz env with dev packages
+        run: mamba env update -n mirflowz -f environment.dev.yml
         
       - name: Run test for rule graph
         run: bash test/test_rule_graph.sh
       
+      - name: Run test for snakemake lint
```

Comment:
"Run linting" or "Run snakemake lint" will do. Or if you want to be very explicit: "Check workflow descriptor files for lints"